install_bootstrap

```diff
@@ -249,7 +249,15 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
 #            read -ep $'What is the email address for notification? \n' email_address
 #      fi
 
-      
+      project_name='project_name'
+      ask_question "What is the name of the project for this instance (e.g. Stage, Commons, AnVIL, etc.)? This name will be used for error messages." "$PROJECT_NAME" "project name" $project_name
+
+      contact_email='contact_email'
+      ask_question "What is a contact email in case there are problems authenticating?" "$CONTACT_EMAIL" "contact email" $contact_email
```

Comment:
I would say `authorizing` instead of `authenticating`.

Reply:
Good catch!